cds-snc/cppd-medical-report
# Translated start section, intro to the medical report

# Description

[Trello card:](https://trello.com/c/a46aCfHK/309-people-can-access-prototype-in-en-and-fr) Translates the start section to the application.

# Test Instructions

1. Go to the application
1. If your browser is in EN-xx locale, click `Français` at the top
1. See French translation (original EN and FR forms are linked in the Trello card)

# Other Notes

There's text that says write stuff at the top of every page, which is obviously no longer applicable and is a different scope than this task. We can change that in a later pull request / [Trello task](https://trello.com/c/WqJVeqsh/356-misc-content-tasks). That might be a good first real change to the medical report content as practice PR reviews with ESDC folks.
